### DISCUSSION TITLE
Informazione su Annotazione Modificativa

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#732] [Supporto servizi cooperativi] [answered]
Buongiorno.
ho cominciato ad analizzare le Annotazioni.
In quelle modificative è possibile indicare solo se l'intestatario dell'atto cambi nome, cognome o entrambi.
Mi aspettavo anche di trovare la possibilità di indicare nel caso la persona modifichi il sesso. In questo caso come ci dobbiamo comportare?

**Comment by amusarra** [comment, marked as answer]
Salve @annamaraspin 
Nel vostro caso dovreste usare l'annotazione per rettifica.

Cordiali Saluti.
Supporto ANSC.

**Comment by annamaraspin** [comment]
OK, grazie mille
